client/src/index.js

```diff
@@ -7,24 +7,15 @@ socket.addEventListener('open', function (event) {
 });
 
 socket.addEventListener('message', function (event) {
-  var questiondiv = document.createElement ('div');
-  var answer1div = document.createElement ('div');
-  var answer2div = document.createElement ('div');
-  var answer3div = document.createElement ('div');
-  var answer4div = document.createElement ('div');
-  
   var obj = JSON.parse(event.data);
-  questiondiv.innerHTML = obj.question;
-  answer1div.innerHTML = obj.answers[0].answer;
-  answer2div.innerHTML = obj.answers[1].answer;
-  answer3div.innerHTML = obj.answers[2].answer;
-  answer4div.innerHTML = obj.answers[3].answer;
+  appendTextDiv(document.getElementById('main'),obj.question);
+  for (i=0;i<4;i++){
```

Comment:
We don't use for loops anymore.

`obj.answers.forEach((answer) => appendTextDiv(mainDiv, answer));`

Reply:
By the way, 

`(answer) => appendTextDiv(mainDiv, answer)`

is almost the same as 
`function (answer) {
 appendTextDiv(mainDiv, answer);
}`

But the => arrow function style is used in modern JS.